APITesters/SwiftAPITester/SwiftAPITester/StoreProductAPI.swift

```diff
@@ -18,11 +18,12 @@ func checkStoreProductAPI() {
     let subscriptionGroupIdentifier: String? = product.subscriptionGroupIdentifier
     let priceFormatter: NumberFormatter? = product.priceFormatter
     let subscriptionPeriod: SubscriptionPeriod? = product.subscriptionPeriod
-    let introductoryPrice: StoreProductDiscount? = product.introductoryPrice
+    let introductoryPrice: StoreProductDiscount? = product.introductoryStoreProductDiscount
     let discounts: [StoreProductDiscount] = product.discounts
 
     let pricePerMonth: NSDecimalNumber? = product.pricePerMonth
-    let localizedIntroductoryPriceString: String? = product.localizedIntroductoryPriceString
+    let localizedIntroductoryStoreProductDiscountString: String? =
+        product.localizedIntroductoryStoreProductDiscountString
```

Comment:
this one doesn't quite roll off the tongue 😄 
I'm not sure we need to specify `StoreProductDiscount` here, though, since the string will only represent the `price` here, so I feel like the previous name would still work. 
for `introductoryPrice` it's different because the previous name made it sound like it was a "price", which I'd expect as a number or a struct with number and currency, and it's slightly more complex than that